### ISSUE TITLE
Installed applications are not shown

### ISSUE THREAD

**Issue by oop7** [inotia00/ReVanced_Extended#1357]
### Type

Other

### Tools used

ReVanced Manager

### Application

I have tried several times to close and open it and nothing happens
1.9.5

### Bug description

Installed applications are not shown

### Steps to reproduce

No preview 

### Relevant log output

```shell
No log
```

### Screenshots or videos

![Screenshot_20230909-155755](https://github.com/inotia00/ReVanced_Extended/assets/110548351/674c3cac-fadf-4449-9f16-7644217e5037)

### Device Environment

Android 13

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.
- [X] I have written the title and contents in English.
- [X] This issue does not reproduce on unpatched YouTube or YT Music.

**Comment by girshcatt** [comment]
In every rvx manager changelog inotia write: RVX Manager is provided simply for compatibility with my patch, if you run into issues follow the other patch method.
This mean that u can open issue only is something crash or not patch, if u need to patch installed application root method follow other patch methods.
To not worry inotia one more time just close the issue

**Comment by oop7** [comment]
Rvx manger the same problem I follow the guide root and must appear YouTube to patch

**Comment by oop7** [comment]
I didn't author anything, check out the video for yourself @inotia00 
https://github.com/inotia00/ReVanced_Extended/assets/110548351/d467d244-5559-43ca-aec4-47f8daf31bcd

**Comment by inotia00** [comment]
> I didn't author anything, check out the video for yourself @inotia00 https://github.com/inotia00/ReVanced_Extended/assets/110548351/d467d244-5559-43ca-aec4-47f8daf31bcd

refer https://github.com/inotia00/ReVanced_Extended/issues/1331#issuecomment-1708859026

**Comment by inotia00** [comment]
Does this issue still persist?
It's been a while since I installed rvx manager, and it loads the installed app list normally.

**Comment by oop7** [comment]
The problem is that when a third of the slots are full, the applications will not appear. For example, when installing yt, yt music, and Reddit, the applications will not display. The solution is to add a fourth patch for an unpopular application or add a fourth empty slot.

![Screenshot_20230913-093334_Pixel_Launcher~4.png](https://github.com/inotia00/ReVanced_Extended/assets/110548351/2811e870-7a83-43b8-998d-24fc4f24bf80)

![Screenshot_20230913-093421~2.png](https://github.com/inotia00/ReVanced_Extended/assets/110548351/e760dbc2-2bd7-4d78-887b-7ab33b18436c)

![Screenshot_20230913-093635_Settings~2.png](https://github.com/inotia00/ReVanced_Extended/assets/110548351/9c598c29-6d9a-4276-be1d-2099e6285ee9)

![Screenshot_20230913-093841_RVX_Manager~2.png](https://github.com/inotia00/ReVanced_Extended/assets/110548351/e7430abd-3495-4f07-8f3a-dba67698c83a)

**Comment by inotia00** [comment]
okay, I'll check it later.

**Comment by inotia00** [comment]
I am not using RVX Manager as my main tool, and I am not good at handling flutter, so when I first released RVX Manager, I declared that I would not fix any issues that users encounter.
(Because there are many other tools to build RVX besides RVX Manager)

But @oop7, thanks to you for providing very specific steps to reproduce the error, I was able to fix this issue.

It will be reflected in the next release.
